### ISSUE TITLE
Assistant API

### ISSUE THREAD

**Issue by DorijanH** [JZO001/Forge.OpenAI#6]
Hi there,

I see that you have examples for using the service with assistant APIs. Are you still working on it or can you create a release?

Cheers

**Comment by JZO001** [comment]
yes sure, please check the latest version and the playgrounds in the code repository

**Comment by sevzas** [comment]
I can confirm that Assistant API is now supported in Forge.OpenAI - OP can close this issue.
